orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -84,6 +85,31 @@ class SqlExecutionRepository(
 
   private val log = LoggerFactory.getLogger(javaClass)
 
+  init {
+    log.info("Creating SqlExecutionRepository with partition=$partitionName with pool=$poolName")
+
+    withPool(poolName) {
+      jooq.transactional {
+        val record: Record? = jooq.fetchOne(table("partition_name"))
```

Comment:
what happens if there are multiple rows in that table?

Reply:
it should never happen... if it does it will throw, i will wrap the exception into something useful, good catch